modules/nf-core/software/gatk4/applybqsr.nf

```diff
@@ -3,7 +3,7 @@ include { initOptions; saveFiles; getSoftwareName } from './../functions'
 params.options = [:]
 def options    = initOptions(params.options)
 
-process GATK_APPLYBQSR {
+process GATK4_APPLYBQSR {
     label 'memory_singleCPU_2_task'
```

Comment:
Do you already want to add the gatk 4.2.0 update here?

Reply:
Good point, we can switch all to gatk 4.2.0 since we're still working on this branch